src/main/java/com/wavefront/config/ApplicationTags.java

```diff
@@ -0,0 +1,76 @@
+package com.wavefront.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * Application Tags YAML Config.
+ *
+ * @author Sushant Dewan (sushant@wavefront.com).
+ */
+public class ApplicationTags {
```

Comment:
There is a ApplicationTags class in core sdk: `com.wavefront.sdk.common.application.ApplicationTags` which confusing (so I named it `MetadataConfig`). We can make the core sdk class also accept a YAML config acceptable file.